azure-mgmt-compute/src/main/java/com/azure/management/compute/VirtualMachineIdentity.java

```diff
@@ -15,7 +15,7 @@
  * The VirtualMachineIdentity model.
  */
 @Fluent
-public final class VirtualMachineIdentity {
+public class VirtualMachineIdentity {
```

Comment:
Why change the final statements?

Reply:
Azure-core bug. This is the workaround until they fix.